CMakeLists.txt

```diff
@@ -121,6 +121,10 @@ option (USE_STATIC_OSL                      "Use static OpenShadingLanguage libr
 option (WARNINGS_AS_ERRORS                  "Treat compiler warnings as errors"                     ON)
 option (HIDE_SYMBOLS                        "When using gcc, hide symbols not on the public API"    ON)
 
+if (CMAKE_SYSTEM_NAME MATCHES "Linux|FreeBSD")
+    option (USE_VISIBILITY_MAP              "When using gcc, hide symbols not on the public API"    OFF)
```

Comment:
How does that interact with the `HIDE_SYMBOLS` option that has the exact same description?
Edit: I see that you changed the description in a subsequent commit, but my question remains.